services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -233,6 +233,7 @@ async def _start_dynamic_service(
         service_uuid=f"{node_uuid}",
         request_dns=extract_dns_without_default_port(request.url),
         request_scheme=request.headers.get("X-Forwarded-Proto", request.url.scheme),
+        request_user_agent=request.headers.get("User-Agent", ""),
```

Comment:
Just checking ... but are you sure "the puppeteer header" information is in the [User-Agent header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) ? Where is this defined in the code? If so, should we have a way to test that?

I wonder if what you are looking for is rather this other "agent" header https://github.com/ITISFoundation/osparc-simcore/blob/212a68b66b517c4b2133fe27aa0e7dbce992e0a7/packages/service-library/src/servicelib/aiohttp/monitoring.py#L115

Just to be sure!

Reply:
```SIMCORE_USER_AGENT_HEADER: Final[str] = "X-Simcore-User-Agent"```

Reply:
I was misled by the initial request. What SAN says is the correct way to go.